using-timescaledb/limitations.md

```diff
@@ -39,7 +42,10 @@ to distributed hypertables:
   hypertable must be present on the access node and all data
   nodes. This applies also to referenced values.
 - Parallel-aware scans and appends are not supported.
-
+- A consistent restore point for backup/restore across nodes is not 
+  natively provided; care must be taken when restoring individual 
```

Comment:
This might be a bit pedantic, but could give them a heads up to what the problems they might run into here.

```suggestion
  natively provided; care must be taken to ensure consistency when restoring individual 
```